po/da.po

```diff
@@ -42,7 +42,7 @@ msgid ""
 "Mount/unmount filesystems defined in the fstab file with the x-udisks-auth "
 "option"
 msgstr ""
-"Monter/afmonter filsystemer defineret i fstab-filen med tilvalget x-storaged-"
```

Comment:
Alright, I'll drop it from the patch then, and just keep the (unrelated) typo fix in de.po.